### DISCUSSION TITLE
Unsplash

### DISCUSSION THREAD

**Discussion by barraIhsan** [darkreader/darkreader#10365] [Q&A]
![image](https://user-images.githubusercontent.com/57800056/203672465-bc532dd5-6052-4f3a-86e9-5e7bcba3e381.png)
Do I have to revert the Unsplash logo?

Also, about random named class.
Its better to use `svg[aria-labelledby="unsplash-home"]` than `.hic6U` right?
(discussion [#8091](https://github.com/darkreader/darkreader/discussions/8091)).
Because, we don't know if the random named class will be changed if the site updated.

**Comment by Gusted** [comment]
> Do I have to revert the Unsplash logo?

Could you give some context about this?

> Also, about random named class. Its better to use `svg[aria-labelledby="unsplash-home"]` than `.hic6U` right? (discussion [#8091]

Yes it is.

**Reply by barraIhsan** [reply]
Original unsplash logo is black
![image](https://user-images.githubusercontent.com/57800056/203881685-7e536f00-e4c1-49e0-80b8-f6a7a2f497df.png)

**Reply by barraIhsan** [reply]
also, can the favicon be reverted too?
![image](https://user-images.githubusercontent.com/57800056/203882017-099aebc1-95dc-4a74-9302-bb22ee7a96c0.png)
